pandas/core/generic.py

```diff
@@ -749,6 +749,20 @@ def swaplevel(self, i=-2, j=-1, axis=0):
     # ----------------------------------------------------------------------
     # Rename
 
+    # renamer function if passed a dict
```

Comment:
I know you just moved the function, but let's take this opportunity to write a brief docstring.